components/google_drive/actions/create-file-from-template/create-file-from-template.mjs

```diff
@@ -0,0 +1,109 @@
+import googleDrive from "../../google_drive.app.mjs";
+import Mustaches from "google-docs-mustaches";
+
+const MODE_GOOGLE_DOC = 0;
+const MODE_PDF = 1;
+const MODE_GOOGLE_DOC_PDF = 2;
```

Comment:
`MODE_GOOGLE_DOC_PDF` is assigned a value, but never used. I would actually remove these. See suggestions below.
```suggestion
```

Reply:
I use strings instead of ints